plenum/common/txn_util.py

```diff
@@ -120,6 +120,10 @@ def get_req_id(txn):
     return txn[TXN_PAYLOAD][TXN_PAYLOAD_METADATA].get(TXN_PAYLOAD_METADATA_REQ_ID, None)
 
 
+def get_digest(txn):
```

Comment:
Please add test for `get_digest` to `test_txn_general_access_utils`

Reply:
done